lib/ffmpegCommandService.js

```diff
@@ -374,13 +374,13 @@ class FfmpegCommandService {
         command.video.forEach((video,i) => {
             filter += ` [${i}:v]`; // start
 
+            // Scale videos to the same size (setsar=1 recommended to coax aspect ratios, may not be needed anymore)
+            filter += ` scale=${outWidth}:${outHeight}:force_original_aspect_ratio=decrease, setsar=1,`;
+
             // Force padding for videos not in the correct aspect ratio ih=inputHeight, oh=outputHeight, etc
             // See examples https://ffmpeg.org/ffmpeg-filters.html#pad-1
             filter += ` pad=ih*${ratio}/sar:ih:(ow-iw)/2:(oh-ih)/2,`;
 
-            // Scale videos to the same size (setsar=1 recommended to coax aspect ratios, may not be needed anymore)
```

Comment:
Looks like it wasn't needed anymore. Heh

Reply:
So I'm actually not sure if its needed... I need to revisit. I'll make it a todo.